content/docs/reference/neon-cli.md

```diff
@@ -0,0 +1,160 @@
+| [--version](#version)   | Show the Neon CLI version number                 | boolean| -                                 |
+| [--help](#help)      | Show the Neon CLI help                           | boolean| -                                 |
+| [-o, --output](#output)| Set the Neon CLI output format                   | string | table                           |
+|             | Possible choices: "json", "yaml", "table" | |                                  |
+| [--api-host](#api-host)  | The API host                        | -      | https://console.neon.tech/api/v2|
+| [--config-dir](#config-dir)| Path to the Neon CLI configuration directory            | string | `/home/<user>/.config/neonctl`   |
+| [--oauth-host](#oauth-host)| URL for the Neon OAuth host              | -      | https://oauth2.neon.tech        |
+| [--client-id](#client-id) | OAuth client id                     | string | neonctl                         |
+| [--api-key](#api-key)   | Neon API key                             | string | ""                                |
+| [--analytics](#analytics) | Enable analytics                    | boolean| true                              |
+
+### --version
+
+  Shows the Neon CLI version number.
+
+  ```bash
+  $ neonctl --version
+  1.9.4
+  ```
+
+### --help
+
+  Shows the `neonctl` command-line help. You can view help for `neonctl` or a `neonctl` command.
+
+  ```bash
+  neonctl --help
+  ```
+
+  To view the options supported by a particular command:
+
+  ```bash
+  neonctl <command> --help
+  ```
+
+### --output
+
+Sets the output format. Supported options are `json`, `yaml`, and `table`. The default is `table`. Table output may limited to a certain number of columns. The `json` and `yaml` output formats show all output.
+
+```bash
+neonctl me --output json
+```
+
+### --api-host
+
+Sets the Neon API host. The default setting is `https://console.neon.tech/api/v2`, which is the [base URL](https://api-docs.neon.tech/reference/getting-started-with-neon-api#neon-api-base-url) for Neon's public API. Generally, you do not have to specify this option unless directed to do so by Neon.
+
+```bash
+neonctl projects list --api-host https://console.neon.tech/api/v2
+```
+
+### --config-dir
+
+Specifies the path to the `neonctl` configuration file. To view the default configuration directory containing you `credentials.json` file, run `neonctl --help`. The credentials file is created when you authenticate using the `neonctl auth` command. This option is only necessary if you move your `neonctl` configuration file to a location other than the default.
+
+```bash
+neonctl projects list --config-directory /home/dtprice/.config/neonctl
+```
+
+### --oauth-host
+
+Specifies the URL of Neon OAuth host used to authenticate your neonctl client to Neon. The default is `https://oauth2.neon.tech`. You should need to configure this setting when using the `neonctl` CLI client.
```

Comment:
Users actually are not suppose to change this, this option exists to be able to write tests for auth and to connect to our staging env